config/locales/en.yml

```diff
@@ -1385,6 +1385,7 @@ en:
         manager: "Protocol management"
         edit: "Edit protocol"
     protocols_io_import:
+      too_long: "... Text is too long so we had to cut it off."
```

Comment:
Can you add the bootstrap html class `<span class="label label-warning"></span>` to this text, so the text is going to be better visible.
https://getbootstrap.com/docs/3.3/components/#labels